app/controllers/api/mentoring/previous_discussions_controller.rb

```diff
@@ -0,0 +1,17 @@
+module API
+  class Mentoring::PreviousDiscussionsController < BaseController
+    def index
+      discussions = ::Mentor::Discussion::Retrieve.(
+        current_user,
+        :finished,
+        page: params[:page]
+      )
```

Comment:
I don't think this is right. This will return all previous discussions, not just those between a mentor and a student?

Reply:
Yep! This is just a stub for now. Can you replace it to have the correct functionality? I'm also unsure whether we need a separate controller for this.